### ISSUE TITLE
Add a new status for employee salary paiement

### ISSUE THREAD

**Issue by lomamech** [Third-Culture-Software/bhima#6376]
Add a new Status for employees who have a negative net salary, this situation will appear for employees who have advances, installment and loans greater than the gross salary

**Comment by jniles** [comment]
@lomamech is this still needed in your opinion?

**Comment by lomamech** [comment]
> Add a new Status for employees who have a negative net salary, this situation will appear for employees who have advances, installment and loans greater than the gross salary

Yes I think it's worth it, a new Status might be needed, but I remember adding algorithms to prevent this situation, tell me if this kind of situation keeps happening

**Comment by jniles** [comment]
So far, I haven't heard any complaints from users, but I also haven't studied it too closely.  I will take a look!

**Comment by jniles** [comment]
I think this issue can be closed.  It appears that there is some code in the multiple payroll to deal with employees with negative salaries.  

https://github.com/Third-Culture-Software/bhima/blob/master/client/src/modules/multiple_payroll/multiple_payroll.js#L205
